homeassistant/components/homewizard/quality_scale.yaml

```diff
@@ -65,7 +69,10 @@ rules:
   entity-translations: done
   exception-translations: done
   icon-translations: done
-  reconfiguration-flow: done
+  reconfiguration-flow:
+    status: todo
+    comment: |
+      Reauthorization for v2 API token revocation is not implemented.
```

Comment:
what about the reauth flow? Can the token expire?

Reply:
I planned to create a reauth flow in a follow-up PR to keep this PR a bit concise. Now I read this again I think we should mark the `repair-issues` as todo instead this one.

There are not (re)configuration options available for HomeWizard